docs/cfn-template.yaml

```diff
@@ -37,6 +37,11 @@ Resources:
           - aws.health
         detail-type:
           - AWS Health Event
+        detail:
+          source:
```

Comment:
The change to the README earlier this year uses the key `service`, but this change refers to `source`. Which one is correct? Can we make sure both files agree?